# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../uclibc/fix_termios_maxbaudrate.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

since SVN r14193 baud rates above 115200 are supported
by libc/termio/cfsetspeed.c

  -- Michael Tross <michael@tross.org>


--- uClibc-20060830/libc/sysdeps/linux/common/bits/termios.h.orig	2006-08-30 09:10:24.000000000 +0200
+++ uClibc-20060830/libc/sysdeps/linux/common/bits/termios.h	2006-11-02 13:56:52.000000000 +0100
@@ -156,7 +156,6 @@
 #endif
 #define  B57600   0010001
 #define  B115200  0010002
-#if 0 /* limited on uClibc, keep in sync w/ cfsetspeed.c */
 #define  B230400  0010003
 #define  B460800  0010004
 #define  B500000  0010005
@@ -171,9 +170,6 @@
 #define  B3500000 0010016
 #define  B4000000 0010017
 #define __MAX_BAUD B4000000
-#else
-#define __MAX_BAUD B115200
-#endif
 #ifdef __USE_MISC
 # define CIBAUD	  002003600000		/* input baud rate (not used) */
 # define CMSPAR   010000000000		/* mark or space (stick) parity */
